addons/binding/org.openhab.binding.opendaikin/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: OpenDaikin Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.opendaikin;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.5.0.qualifier
+Import-Package: 
+ javax.measure.quantity,
+ javax.ws.rs.client,
```

Comment:
You cannot assume that these packages are available at the client, please use the default provided Jetty, see also:

https://www.openhab.org/docs/developer/development/guidelines.html#c-language-levels-and-libraries

> To allow optimized JavaSE 8 runtimes, the set of Java packages to be used is furthermore restricted to [Compact Profile 2](https://www.oracle.com/technetwork/java/embedded/resources/tech/compact-profiles-overview-2157132.html)

CC: @openhab/2-x-add-ons-maintainers